src/core/public/_variables.scss

```diff
@@ -4,3 +4,7 @@ $osdHeaderOffset: $euiHeaderHeightCompensation;
 $osdHeaderBreadcrumbBlueBackground: #b9d9eb;
 $osdHeaderBreadcrumbGrayBackground: #d9e1e2;
 $osdHeaderBreadcrumbCollapsedLink: #002a3a;
+$osdHeaderPacificSkyDarkestBackground: #163f66;
+$osdHeaderMidnightSkyMediumBackground: #4c636f;
+$osdHeaderMidnightSkyMediumLightColor: #96a0a5;
+$osdMidnightSkyMediumLightHoverColor: #b0b8bb;
```

Comment:
These names aren't consistent with the light mode color names.

Reply:
let me fix that!